lib/libarbiter/player.cc

```diff
@@ -37,36 +39,29 @@ Player* NewPlayerCSV(std::string player_string) {
   } catch (std::invalid_argument err) {
     BOOST_LOG_TRIVIAL(warning)
         << "Player CSV data contained invalid ELO. Data: " << player_string;
-    return nullptr;
+    throw std::invalid_argument("Player CSV data contained invalid ELO.");
   }
-  return new Player(player_name, player_elo);
+  return Player(player_name, player_elo);
```

Comment:
The only reason I made it a pointer was so that I could return `nullptr` if an error occurred. Since I changed it to throw an exception on failure, I don't need that anymore.